lighthouse-plugin-publisher-ads/audits/ad-blocking-tasks.js

```diff
@@ -70,7 +71,7 @@ const HEADINGS = [
 class AdBlockingTasks extends Audit {
   /**
    * @return {LH.Audit.Meta}
-   * @override
+   * /override This member cannot have a JSDoc comment with an '@override' tag because its containing class ... does not extend another class.
```

Comment:
Can we remove these comments?

Reply:
Yeah, I'll remove the entire line.

Reply:
BTW I'm not sure why this error occurs now, but it didn't seem worth the effort to figure it out.